### ISSUE TITLE
daemon.php log is getting bigger every second?

### ISSUE THREAD

**Issue by DamirPecnik** [Block8/PHPCI#793]
I have a problem i set the daemon to run my builds but the log file is being populated every few seconds and no builds are issued is this normal or is a bug because in no time the log file is in GB size big?

It writes this:
.[2015-02-11 19:40:04] DaemoniseCommand.INFO: Finding builds to process [] []
[2015-02-11 19:40:04] DaemoniseCommand.INFO: Found 0 builds [] []
[2015-02-11 19:40:04] DaemoniseCommand.INFO: Finished processing builds. [] []
.[2015-02-11 19:40:09] DaemoniseCommand.INFO: Finding builds to process [] []
[2015-02-11 19:40:09] DaemoniseCommand.INFO: Found 0 builds [] []
[2015-02-11 19:40:09] DaemoniseCommand.INFO: Finished processing builds. [] []
.[2015-02-11 19:40:15] DaemoniseCommand.INFO: Finding builds to process [] []
[2015-02-11 19:40:15] DaemoniseCommand.INFO: Found 0 builds [] []
[2015-02-11 19:40:15] DaemoniseCommand.INFO: Finished processing builds. [] []
.

**Comment by tvbeek** [comment]
I never take a look to the log file, I see that it is cleared after a restart. And because it at least once a week I never had a problem with it. But my log file is deamon.log and not .php Are you sure that you have a .php file with the log?
Maybe it is a good idea to create a cleanup option for it.

**Comment by DamirPecnik** [comment]
I meant daemon.log sorry for misspelling.

**Comment by tvbeek** [comment]
Is your log file really a size of GB? Because I only have a couple of mb for a week (less then 50)
We still need to have a way to change this. Maybe we can logrotate for this part.

**Comment by DamirPecnik** [comment]
Yeah its in GB as it writes every 5 seconds that line above! I have disabled it and using cron for the Jobs!

**Comment by mavimo** [comment]
Can you try to comment [line 103 in RunCommand class](https://github.com/Block8/PHPCI/blob/master/PHPCI%2FCommand%2FRunCommand.php#L103) and check if this "solve" your issue?
